doc/stdlib/base.rst

```diff
@@ -156,6 +156,10 @@ Getting Around
 
    This is only needed if your module depends on a file that is not used via ``include``\ . It has no effect outside of compilation.
 
+.. function:: apropos(string)
+
+   .. Docstring generated from Julia source
+
```

Comment:
You don't need the `.. Docstring generated from Julia source` line, it is added automatically. What's the warning genstdlib gives about missing apropos (which module)? Are you sure you are running it with the version of julia compiled with the doc?